README.md

```diff
@@ -0,0 +1,24 @@
+# TODO Add a readme
+
+Here's an example
+https://gist.github.com/PurpleBooth/109311bb0361f32d87a2
+
+
+# Questions
+
+- How do I visit the end points? Is the url localhost:3000?
+- How do I hit one of the urls? for `\user`, how do I get a response? can you provide an example in the readme?
+- What is the difference between `var`, `const`, `let`? Can you update the code accordingly to use the proper es6 conventions?
+(Hint : https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75)
+- What version of the node.js are you using? Can you add an engine to the package.json?
+(hint : https://docs.npmjs.com/files/package.json#engines )
+
+
+# Next Steps with Twitter
+
+- To access twitter, use this node package: https://www.npmjs.com/package/twitter, can you implement this?
```

Comment:
I looked at the Node.js twitter package and I saw it hides a lot of complexity from me (i.e. getting OAuth token) and for that reason, for now, I will implement everything by hand so I can learn this low level interactions, and will switch to higher level library after I gain complete understanding of how it works.